### ISSUE TITLE
Globbing for Sass partials

### ISSUE THREAD

**Issue by davetgreen** [mkdo/kapow-sass#11]
To shave off a few more seconds during development, it would be useful if globbing could be introduced to reduce the need for adding new partials to the main `site.scss` file.

This would need to exclude settings/plumbing/config folders which need to be particular when it comes to loading order. Recommend keeping this to component, module and page folders etc.

**Comment by davetgreen** [comment]
This is something that requires some work over in Kapow Grunt to make the globbing happen.

**Comment by davetgreen** [comment]
On hold until the globbing has been introduced in the Grunt module.

**Comment by davetgreen** [comment]
This is now unblocked now that `grunt-sass-globbing` has been introduced into Kapow! Grunt

**Comment by davetgreen** [comment]
Completed.
